### PR TITLE
Update openstack_collector.py

### PR DESCRIPTION
**Pull request by markchristopherwest**
Additional Log Support:

log_dirs = /var/log/ceilometer, /var/log/cinder, /var/log/glance, /var/log/heat, /var/log/keystone, /var/log/nova, /var/log/neutron, /var/log/openvswitch

Additional Feature Support:

hardware collection

### CODE REVIEW

File: openstack_collector.py [hunk 1]
```diff
@@ -90,7 +91,16 @@ class Node(object):
         'iscsi_session': ('iscsiadm -m session', False),
         'fc_hba_info': ('systool -c fc_host -v', False),
         'disk_by_path': ('ls -l /dev/disk/by-path', False),
-        'multipath_ll': ('multipath -ll', True)
+        'multipath_ll': ('multipath -ll', True),
+        'ip_a': ('ip a', False),
+        'platform_os': ('cat /etc/redhat-release', True),
```
Comment (denny-zhao) [inline openstack_collector.py:96]: Looks like it's redhat specific. Can it be more general?

File: openstack_collector.py [hunk 2]
```diff
@@ -90,7 +91,16 @@ class Node(object):
...
+        'platform_kernel': ('uname -r', True),
+        'lspci': ('lspci', True),
+        'lshw': ('lshw', True),
+        'dmidecode': ('dmidecode', True),
+        'uptime': ('uptime', False),
+        'os_service_status': ('openstack-service status', False),
```
Comment (peter-wangxu) [inline openstack_collector.py]: osp specific?

Comment (peter-wangxu) [inline openstack_collector.py:102]: Still osp specific?

File: openstack_collector.py [hunk 3]
```diff
@@ -90,7 +91,16 @@ class Node(object):
...
+        'os_status': ('openstack-status', False)
```
Comment (peter-wangxu) [inline openstack_collector.py:103]: Ditto

**Comment by markchristopherwest** [comment]
Resolved

**Pull request closed without merging**